Library/Homebrew/api/internal.rb

```diff
@@ -41,10 +41,11 @@ def self.formula_stub(name)
         end
 
         stub = Homebrew::FormulaStub.new(
-          name:        name,
-          pkg_version: PkgVersion.parse(stub_array[0]),
-          rebuild:     stub_array[1],
-          sha256:      stub_array[2],
+          name:           name,
+          pkg_version:    PkgVersion.parse(stub_array[0]),
+          version_scheme: stub_array[1],
+          rebuild:        stub_array[2],
```

Comment:
Should consider omitting these fields when they are `0` if it makes any meaningful reduction to space.